vendor/src/github.com/microsoft/hcsshim/createprocess.go

```diff
@@ -43,7 +86,7 @@ func CreateProcessInComputeSystem(id string, params CreateProcessParams) (proces
 	if err != nil {
 		err = fmt.Errorf(title+" - Failed conversion of id %s to pointer %s", id, err)
 		logrus.Error(err)
-		return 0, err
+		return
```

Comment:
Okay apparently Go handles shadowing properly and errors out, so disregard my comment